sway-lib-std/src/math.sw

```diff
@@ -193,15 +211,65 @@ impl BinaryLogarithm for u8 {
     }
 }
 
+impl BinaryLogarithm for u256 {
```

Comment:
Please move this implementation above the implementation for u64

Reply:
@tritao Is this particular linearity example something we should have fixed already or something we need the collection context for?

Reply:
Are we waiting on #5572 to close before we merge this then?